connectors/sources/outlook.py

```diff
@@ -771,6 +773,14 @@ def get_default_configuration(cls):
                 "ui_restrictions": ["advanced"],
                 "value": False,
             },
+            "max_concurrent_tasks": {
+                "default_value": MAX_CONCURRENCY,
+                "display": "numeric",
+                "label": "Maximum concurrent tasks",
+                "order": 8,
```

Comment:
There is already `"order": 8`, and it may not be displayed as expected when `Outlook Server` is selected.

Reply:
Oh i see. Actually the above config field `use_text_extraction_service` was at 7th order so i thought this can be at 8th. I'll update the order for both of them.